bin/jsonnet-lint

```diff
@@ -7,4 +7,5 @@ DOCKER_USER="$(id -u):$(id -g)"
 # the dashboard directory is mounted in the container working directory, e.g.
 # src/dashboards is mounted in /app/dashboards (with /app as the working
 # directory).
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm app jsonnet-lint "${@//src\//}"
+COMMAND="for file_name in ${@//src\//}; do jsonnet-lint \$file_name ; done"
+DOCKER_USER=${DOCKER_USER} docker-compose run --rm app /bin/sh -c "$COMMAND"
```

Comment:
```suggestion
DOCKER_USER=${DOCKER_USER} docker-compose run --rm app /bin/sh -c "${COMMAND}"
```

Reply:
In the meantime, add a "FIXME" mentionning that we are waiting this PR to have been merged.

Reply:
Thanks) Updated with the fixme comment and the `"${COMMAND}"`